### DISCUSSION TITLE
`denoify.config.js` can be type checked

### DISCUSSION THREAD

**Discussion by marcomontalbano** [garronej/denoify#83] [Ideas]
Hi, I would like to suggest writing the `denoify.config.js` file in the following way. This could be part of the documentation.

```js
// @ts-check

/** @type { import('denoify/lib/config/parseParams').DenoifyParams } */
const config = {

}

module.exports = config
```
![denoify.config.js](https://user-images.githubusercontent.com/1681269/210899134-4409368a-1997-487e-86c1-3a9b4f7b1ee8.gif)

**Comment by garronej** [comment]
Yeah <3 
I'll update the doc

**Comment by garronej** [comment]
DONE: https://docs.denoify.land/build-options  

I love this!

**Comment by marcomontalbano** [comment]
That's great!

Two missing things:
1. `module.exports = config` is missing at the end of the file.
2. In that page there's a typo. `denoifyrc.config.js` (and `denoifyrc.config.json`) should be without the suffix `rc`.
    > *Denoify be looking at the root of your project for a `denoifyrc.config.js` (or a `denoifyrc.config.json`) configuration file.*

**Reply by garronej** [reply]
Oh boy...  
Thank you!!, I fixed it.
